.github/CONTRIBUTING.md

```diff
@@ -49,6 +67,9 @@ outlined on that page and do not file a public issue.
 * 4 spaces for indentation rather than tabs
 * Public classes and methods must contain header documentation
 * Use plain C functions whenever possible (as opposed to class methods)
+* Restrict subclassing (objc macro, final in examples)
+* instance variables instead of properties
+* local variables instead of self.prop over & over
```

Comment:
Capitalization again, also how about 

> Create local variables instead of accessing properties like `self.property` repeatedly. This results in a larger binary and extra `objc_msgSend(...)` calls.